cmd/minikube/cmd/start.go

```diff
@@ -336,6 +338,14 @@ func clusterUp(config *cluster.MachineConfig) {
 	}
 }
 
+func validateOpenshiftVersion() {
+	if viper.IsSet(openshiftVersion) {
+		if !util.ValidateOpenshiftMinVersion(viper.GetString(openshiftVersion)) {
+			glog.Exitf("%s is not supported with minishift", viper.GetString(openshiftVersion))
```

Comment:
Let's use fmt.Println. For a user facing message we don't want to have all this log gibberish. I guess we could see how to configure glog, but we need a review of the whole logging approach anyways.